### DISCUSSION TITLE
question about Fundamentals of Redux Course from Dan Abramov on egghead.io

### DISCUSSION THREAD

**Discussion by chenjuexu** [reduxjs/redux#4102] [Help]
I am learning redux and mock the teacher's code,I just change a little,but I do not know why it can not work,and I want to try plenty of ways to adjust it,but it is too little too try and nowhere could start.Below is the code causing bug I guess because the project could not work while I write a little in it.The below is almost 100% with Dan Abramov's versionhttps://embed.plnkr.co/github/eggheadio-projects/getting-started-with-redux/master/23-react-redux-extracting-container-components-visibletodolist-addtodo?show=script,preview
```
const Deeplists = ({ store }) => {
  const [_, forceUpdate] = React.useReducer((x) => x + 1, 0);
  store.subscribe(forceUpdate);
  return (
    <Lists
      todos={deepertodos(
        store.getState().todos,
        store.getState().visibilityFilter
      )}
      onTodoClick={id=> store.dispatch({ type: "TOGGLE_TODO", id })}
    />
  );
};
```
This is my project link:https://codepen.io/nutkin/pen/PopOamN?editors=0010

**Comment by markerikson** [comment]
Unfortunately, this is not a good place to ask help questions. For usage questions (which is what I believe this is), please use Stack Overflow or Reactiflux where there are a lot more people ready to help you out. Thanks!
